docker/Dockerfile

```diff
@@ -33,7 +34,9 @@ USER root
 # Overriding the cache directory to install the deps for the Chrome
 # version installed for pptruser. 
 RUN PUPPETEER_CACHE_DIR=/home/pptruser/.cache/puppeteer \
-    npx puppeteer browsers install chrome --install-deps
+    npx puppeteer browsers install chrome --install-deps \
+     && apt-get clean \
```

Comment:
aren't these files going to be cached in the image anyway because they are created in the step on line 13? could you provide the size of the image before and after the change to see how big is the gain?

Reply:
I think what would be better is trying to move the apt-update step into this RUN block

Reply:
I dig up a little and it seem that if we do this suggestion we will save around 20MB.